JSON.java

```diff
@@ -0,0 +1,47 @@
+package com.mapzen.valhalla;
+
+import com.google.gson.annotations.SerializedName;
+
+public class JSON {
+    public Location[] locations = new Location[2];
```

Comment:
The JSON request object still only supports a start and destination. This needs to be modified to support an arbitrary number of locations.